integrations/chronosphere/manifest.json

```diff
@@ -0,0 +1,61 @@
+{
+  "name": "Chronosphere Events",
+  "version": "1.0.0",
+  "overview": "Monitor LaunchDarkly feature flag changes in Chronosphere.",
+  "description": "Monitor LaunchDarkly feature flag change events in Chronosphere.",
+  "author": "LaunchDarkly",
+  "supportEmail": "support@launchdarkly.com",
+  "links": {
+    "site": "https://chronosphere.io/",
+    "launchdarklyDocs": "https://docs.launchdarkly.com/integrations/chronosphere",
+    "privacyPolicy": "https://chronosphere.io/privacy/"
+  },
+  "categories": ["monitoring"],
+  "icons": {
+    "square": "assets/square.svg",
+    "horizontal": "assets/horizontal.svg"
+  },
+  "formVariables": [
+    {
+      "key": "secretToken",
+      "name": "Chronosphere secret token",
+      "type": "string",
+      "description": "Enter your Chronosphere secret token.",
+      "isSecret": true
+    },
+    {
+      "key": "receiverURL",
+      "name": "Chronosphere receiver URL",
+      "description": "Your Chronosphere receiver URL. Should look like https://{MY_COMPANY}.chronosphere.io/api/v1/data/events/receiver/launchdarkly",
+      "type": "string"
+    }
+  ],
+  "capabilities": {
+    "auditLogEventsHook": {
+      "endpoint": {
+        "url": "{{receiverURL}}",
+        "method": "POST",
+        "headers": [
+          {
+            "name": "Content-Type",
+            "value": "application/json"
+          },
+          {
+            "name": "X-LD-Signature",
+            "value": "{{secretToken}}"
+          }
+        ]
+      },
+      "templates": {
+        "default": "templates/default.json.hbs"
+      },
+      "defaultPolicy": [
+        {
+          "effect": "allow",
+          "resources": ["proj/*:env/production:flag/*"],
+          "actions": ["*"]
```

Comment:
@sophia-hanley I'll be pulling this manifest locally to test it but wanted to mentioned with this wildcard for actions, you'll end up receiving a number of change events that are probably not what you're interested in. For example things like tags being added/removed from a flag. They'll have no affect on the runtime.

You can see our suggested list here: https://docs.launchdarkly.com/guides/integrations/build-audit-log-event-hook/?q=building#manifestjson-example though you could even drop the segment section from it.

Reply:
great, I took a look and added the specific actions we're interested in. @InTheCloudDan did this work when you ended up pulling it locally?